eZ/Bundle/EzPublishCoreBundle/Command/ReindexCommand.php

```diff
@@ -239,7 +239,7 @@ protected function indexIncrementally(InputInterface $input, OutputInterface $ou
         $progress = new ProgressBar($output);
         $progress->start($iterations);
 
-        if ($processCount > 1) {
+        if ($processCount > 1 && !$continueOnError) {
```

Comment:
you should probably take this into account earlier, otherwise `$processMessage` above will be wrong.